components/views/auth/Login.tsx

```diff
@@ -142,9 +143,9 @@ const Login = () => {
                             className={styles.button}
                             type="submit"
                             disabled={disableSubmit}>
-                            Submit
+                            {user?.loading ? <CircleLoader /> : 'Submit'}
                         </button>
-                        <button onClick={onBanner}>BANNER</button>
+                        <button onClick={onBanner}>WTF IS THIS BANNER</button>
```

Comment:
I missed this. dumb. you left it in here, dumb. We are a pair of dumb dumbs. remove my trash por favor

Reply:
gladly